packages/next/src/server/config-shared.ts

```diff
@@ -567,6 +567,12 @@ export interface ExperimentalConfig {
    * Enables the new dev overlay.
    */
   newDevOverlay?: boolean
+
+  /**
+   * @experimental This is temporary and will be **removed** in the short future.
+   * Enables the streaming metadata by default or not.
```

Comment:
I think we should make the language a bit more consistent with the other flags above 

```suggestion
   * When enabled will cause async metadata calls to stream rather than block the render. 
```

Reply:
Seemed redundant since we already have actual experimental warnings when running the dev server & we don't do it for other flags, but I don't have a strong opinion here.

Reply:
Kinda wanna inform users that this is not like other experimental flag for experimental feature but more like a temporary flag

Reply:
Well all flags are experimental right? Until they're stable

Reply:
makes sense, dorpped it